src/HttpServer.ts

```diff
@@ -93,6 +95,27 @@ export class HttpServer {
         res.send(status);
     }
 
+    /**
+     * @openapi
+     * /getaddr:
+     *   get:
+     *     summary: It retrieves some useful information about the server status.
+     *     description: It shows the information about the server status.
+     *     parameters:
+     *       - in: path
+     *         name: verifier
+     *         required: false
+     *         description: The address of the verifier (Not used at the moment).
+     *         schema:
+     *           type: string
```

Comment:
It looks like this parameter isn't used at all: https://github.com/antomor/rif-relay-server/blob/94ebf465235c31da711ff681bbba45694fe5202b/src/RelayServer.ts#L126

Reply:
I agree with your proposal; created [a task](https://rsklabs.atlassian.net/browse/PP-125) for this.